openreview/conference/templates/recruit_reviewers_pre_process.py

```diff
@@ -13,4 +14,9 @@ def process(client, note, invitation):
     if not client.get_groups(id=REVIEWERS_INVITED_ID, member=user):
         raise openreview.OpenReviewException('User not in invited group, please accept the invitation using the email address you were invited with')
 
+    if note.content['response'] == 'No':
+        memberships = client.get_groups(regex=VENUE_ID+'/Paper.*/Reviewers$', member=user)
```

Comment:
a small suggestion to avoid having the string `Reviewers` here, you can use:

`conference.get_reviewers_id(number='.*')` as a constant in the template and then use that constant in the regex.

Reply:
Done! I also realized that this preprocess function is also used for example when we use Assignment_Recruitment. In this case I believe we want users to be able to decline (as they are removed from paper anon group) so I added a check. Let me know if I am missing anything!